contracts/non-fungible-tokens/andromeda-crowdfund/src/contract.rs

```diff
@@ -87,6 +92,16 @@ pub fn reply(_deps: DepsMut, _env: Env, msg: Reply) -> Result<Response, Contract
 
 #[cfg_attr(not(feature = "library"), entry_point)]
 pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
+    let prev = get_contract_version(deps.storage)?;
```

Comment:
Could we make this process a shared method in the `std` crate?

Something like `ensure_from_min_version(deps.storage, "1.1.0")?`

Reply:
Defined a shared method